draft-ietf-quic-transport.md

```diff
@@ -2763,26 +2792,55 @@ Note:
   congestion control, which are not expected to be relevant for a closed
   connection. Retransmitting the final packet requires less state.
 
-New packets from unverified addresses could be used to create an amplification
-attack; see {{address-validation}}.  To avoid this, endpoints MUST either limit
-transmission of CONNECTION_CLOSE frames to validated addresses or drop packets
-without response if the response would be more than three times larger than the
-received packet.
+While in the closing period, an endpoint could receive packets from a new
+source address, indicating a connection migration; see {{migration}}. New
+packets from unverified addresses could be used to create an amplification
+attack; see {{address-validation}}. To avoid being used for an amplication
+attack, an endpoint in the closing state MUST either discard packets received
+from unvalidated addresses or limit the cumulative size of packets it sends to
+unvalidated addresses to 3 times the size of packets it receives from the
+address.
```

Comment:
This required a little better alignment with the other text above.  The sentence "To avoid being used for an amplification attack" is now the same in both places.